pages/blog/index.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+import { NextPage } from "next";
+import { useState, useEffect, useCallback } from "react";
+import Loader from "../../components/blog/Loader";
+import PageHeader from "../../components/blog/PageHeader";
+import PostList, { Article } from "../../components/blog/PostList";
+import Footer from "../../components/Footer";
+import { fetchAPI } from "../../utils/blog/fetch-api";
+import Navbar from "../../views/Navbar";
+
+
+interface Meta {
+	pagination: {
+		start: number;
+		limit: number;
+		total: number;
+	};
+}
+const Profile: NextPage = () => {
+	const [meta, setMeta] = useState<Meta | undefined>();
+	const [data, setData] = useState<Article[]>([]);
+	const [isLoading, setIsLoading] = useState(true);
+
+	const fetchData = useCallback(async (start: number, limit: number) => {
+		setIsLoading(true);
+		try {
+			const path = `/articles`;
+			const urlParamsObject = {
+				sort: { createdAt: "desc" },
+				populate: {
+					cover: { fields: ["url"] },
+					category: { populate: "*" },
+					authorsBio: {
+						populate: "*",
+					},
+				},
+				pagination: {
+					start: start,
+					limit: limit,
+				},
+			};
+			const responseData = await fetchAPI(path, urlParamsObject);
+
+			if (start === 0) {
+				setData(responseData.data);
+			} else {
+				setData((prevData: Article[] ) => [...prevData, ...responseData.data]);
+			}
+
+			setMeta(responseData.meta);
+		} catch (error) {
+			console.error(`[blog/index.tsx] Unable to fetch articles - ${JSON.stringify(error)}`);
+		} finally {
+			setIsLoading(false);
+		}
+	}, []);
+
+	function loadMorePosts(): void {
+		const nextPosts = meta?.pagination.start ?? 0 + (meta?.pagination.limit ?? 0);
+		fetchData(nextPosts, Number(process.env.NEXT_PUBLIC_PAGE_LIMIT));
+	}
+
+	useEffect(() => {
+		fetchData(0, Number(process.env.NEXT_PUBLIC_PAGE_LIMIT));
+	}, [fetchData]);
+
+	if (isLoading) return <Loader />;
+
+	return (
+		<div>
```

Comment:
Is `div` really necessary there? Can't we just use a `React.Fragment`?

```suggestion
		<>
```

Reply:
done